themes/default/layouts/partials/docs/table-of-contents.html

```diff
@@ -1,63 +1,80 @@
+                        </ul>
+                        {{ if strings.Contains $.Page.Permalink "registry/packages" }}
+                            {{ $path := (split .Page.Params.title_tag ".") }}
+                            {{- if eq (len $path) 3 -}}
+                                {{ $packageName := index $path 0 }}
+                                {{ $moduleName := index $path 1 }}
+                                {{ $resourceName := index $path 2 }}
+                                <div class="mt-6">
+                                    <pulumi-resource-links
+                                        package-name="{{ $packageName }}"
+                                        module-name="{{ $moduleName }}"
+                                        resource-name="{{ $resourceName }}"
+                                    ></pulumi-resource-links>
+                                </div>
+                            {{ end }}
+                        {{ end }}
                     </div>
                 </div>
             </div>
         </div>
-    {{ end }}
-    <ul class="p-0 list-none table-of-contents-feedback">
-        <!-- Only show "Edit this Page" for pages that are not generated. -->
-        {{ $isCLICommand := hasPrefix .Path "docs/cli/commands" }}
-        {{ $isAPIDoc := and (hasPrefix .Path "docs/reference/pkg/") (ne .Path "docs/reference/pkg/_index.md") }}
-        {{ if not (or $isCLICommand $isAPIDoc $.Page.Params.no_edit_this_page) }}
-            <li>
-                <a
-                    data-track="edit-page"
-                    class="text-gray-600 hover:text-gray-700 text-xs"
-                    href="{{ $.Site.Params.contentRepositoryURL }}/edit/{{ $.Site.Params.contentRepositoryBaseBranch }}/{{ getenv "REPO_THEME_PATH" }}content/{{ .Path }}"
-                    target="_blank"
-                >
-                    <i class="fas fa-edit mr-2" style="width: 14px"></i>Edit this Page
-                </a>
-            </li>
-        {{ end }}
+    </div>
+{{ end }}
+<ul class="p-0 list-none table-of-contents-feedback">
+    <!-- Only show "Edit this Page" for pages that are not generated. -->
+    {{ $isCLICommand := hasPrefix .Path "docs/cli/commands" }}
+    {{ $isAPIDoc := and (hasPrefix .Path "docs/reference/pkg/") (ne .Path "docs/reference/pkg/_index.md") }}
+    {{ if not (or $isCLICommand $isAPIDoc $.Page.Params.no_edit_this_page) }}
         <li>
             <a
-                data-track="request-change"
+                data-track="edit-page"
                 class="text-gray-600 hover:text-gray-700 text-xs"
-                href="{{ $.Site.Params.contentRepositoryURL }}/issues/new?body=File: [{{ getenv "REPO_THEME_PATH" }}content/{{ .Path }}]({{ $.Page.Permalink }})"
+                href="{{ $.Site.Params.contentRepositoryURL }}/edit/{{ $.Site.Params.contentRepositoryBaseBranch }}/{{ getenv "REPO_THEME_PATH" }}content/{{ .Path }}"
```

Comment:
this link looks like it opens the code editor in github in the browser with the changes are against master by default in edit mode. the user will need to make their own branch and PR, so not sure if this may end up being confusing.

![image](https://github.com/pulumi/pulumi-hugo/assets/16751381/57215be5-deca-4303-bc6d-422267bdbde6)

Reply:
@sean1588 this new flow in github, when you select that commit button, it goes through a flow of creating a branch; or it should; happy to change this though,  its just what was already in use

Reply:
mind doing a follow on pr though if its an issue?

Reply:
ahh ok, never knew it did that when you select the commit button. that sounds good then! thanks!